packages/babel-preset/index.js

```diff
@@ -110,8 +97,18 @@ module.exports = function shopifyCommonPreset(
     transformRuntime && [
       '@babel/plugin-transform-runtime',
       {
-        ...transformRuntimeOptions,
+        corejs: false,
+        helpers: true,
+        // By default, babel assumes babel/runtime version 7.0.0-beta.0,
+        // explicitly resolving to match the provided helper functions.
+        // https://github.com/babel/babel/issues/10261
         version: require('@babel/runtime/package.json').version,
```

Comment:
Previously version came after `...transformRuntimeOptions` which meant that it was always set. Now the user can override it. I can't ever thing of a reason why they'd want to do that but it feels more flexible than forcing it.